app/src/main/java/com/cdio/solitaire/imageanalysis/CardDataCreationModel.java

```diff
@@ -0,0 +1,65 @@
+package com.cdio.solitaire.imageanalysis;
+
+import android.graphics.Bitmap;
+
+import com.google.android.gms.common.util.ArrayUtils;
+
+import org.opencv.android.Utils;
+import org.opencv.core.Mat;
+import org.opencv.imgproc.Imgproc;
+
+import java.util.Arrays;
+
+public class CardDataCreationModel {
+
+    // Load in OpenCV library
+    static {
+        System.loadLibrary("opencv_java4");
```

Comment:
Nothing else uses opencv in the project?
Otherwise maybe to main?

Reply:
Now moved to mainActivity